src/ShareModal.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+import React from 'react';
+import ModalHeader from './ModalHeader';
+import ModalWithFooter from './ModalWithFooter';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import type {IntlShape} from 'react-intl';
+import {ReactComponent as ShareIcon} from './svg/Share.svg'
+
+import './ShareModal.scss';
+
+type ShareCompleteModalProps = {
+    intl: IntlShape,
+    show: boolean,
+    onCancel: () => void,
+    onConfirm: () => void
+};
+
+class ShareCompleteModal extends React.Component<ShareCompleteModalProps, {}> {
+    static defaultProps = {
+        show: false,
+        onCancel: () => {}
+    }
+
+    copyURL = () => {
+        // Copy the URL to the clipboard, see:
+        // https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText
+        if (navigator.clipboard) {
+            navigator.clipboard.writeText(document.location.href);
+        }
+    }
+
+    componentDidMount() {
+        this.copyURL();
+    }
+
+    /* istanbul ignore next */
+    handleKeyDown = (event: KeyboardEvent) => {
+        if (event.key === ' ' || event.key === 'Enter') {
+            event.preventDefault();
+            this.copyURL();
+        }
+    }
+
+    render () {
+        const buttonProperties = [{
+            label: this.props.intl.formatMessage({id: 'ShareModal.cancel'}),
+            onClick: this.props.onCancel,
+            isPrimary: false
+        }];
+
+        return(
+            <ModalWithFooter
+                show={this.props.show}
+                focusOnOpenSelector={'#ShareModal-copy'}
```

Comment:
This Id doesn't exist, but it's working as expected, because the share button is first focusable item in the modal.

Reply:
Update to use the right selector (class name).